btcjson/chainsvrresults.go

```diff
@@ -117,6 +117,19 @@ type GetBlockVerboseTxResult struct {
 	NextHash      string        `json:"nextblockhash,omitempty"`
 }
 
+// ChainTip is a single chain tip.
+type ChainTip struct {
+	Height    int32  `json:"height"`
+	Hash      string `json:"hash"`
+	BranchLen int32  `json:"branchlen"`
+	Status    string `json:"status"`
+}
+
+// GetChainTipsResult models the data from the getchaintips command.
+type GetChainTipsResult struct {
+	ChainTips []ChainTip `json:"chaintips"`
+}
```

Comment:
I'm not sure how closely you want to follow the Bitcoin Core RPC response. From the [docs](https://github.com/btcsuite/btcd/blob/master/docs/json_rpc_api.md#1-overview) I got the impressions that some RPC calls are/were compatible with Bitcoin Core.

Bitcoin Core's `getchaintips` RPC returns a JSON list of chain tips as top level element:
```
[
  {
     "height": 115861,
     "hash":" 000000407476239a4870e510ce1dea262c9afa672eae3ab50a019812f83e0059",
     "branchlen": 0,
     "status": "active"
  }
]
```

This currently returns a JSON map with a single key called `chaintips` and the list of chain tips as value.


```
"chaintips": [
  {
     "height": 115861,
     "hash":" 000000407476239a4870e510ce1dea262c9afa672eae3ab50a019812f83e0059",
     "branchlen": 0,
     "status": "active"
  }
]
```

Reply:
Changed the output to match that of Bitcoin Core